validate/validate.go

```diff
@@ -228,6 +230,16 @@ func PolicyToOptions(policy *cpb.Policy) (*Options, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	switch policy.Product.Name {
+	case spb.SevProduct_SEV_PRODUCT_TURIN:
+		tcbVersion = kds.TCBStructVersion1
+	case spb.SevProduct_SEV_PRODUCT_MILAN, spb.SevProduct_SEV_PRODUCT_GENOA:
+		tcbVersion = kds.TCBStructVersion0
```

Comment:
Has this been tested with Siena and Bergamo?

Reply:
Thanks for the pointers! I haven't explicitly tested with Sienna and Bergamo, but it looks like we should as they are also part of the Zen4 architecture like Genoa.